docker/c7n

```diff
@@ -1,12 +1,12 @@
 # Dockerfiles are generated from tools/dev/dockerpkg.py
-FROM ubuntu:24.04 as build-env
+FROM ubuntu:24.04 AS build-env
 
 ARG POETRY_VERSION="1.8.3"
 SHELL ["/bin/bash", "-c"]
 
 # pre-requisite distro deps, and build env setup
 RUN apt-get --yes update
-RUN apt-get --yes install --no-install-recommends build-essential curl python3-venv python3-dev adduser
+RUN apt-get --yes install --no-install-recommends build-essential     curl python3-venv python3-dev adduser
```

Comment:
Any idea where the extra spacing came from here? I don't see any change that would have led to it, seems stray

Reply:
its the line continuation in the docker file, if its removed in the dockerpkg.py it triggers a lint warning though.